foxy/distribution.yaml

```diff
@@ -4419,6 +4419,22 @@ repositories:
       url: https://github.com/oKermorgant/slider_publisher.git
       version: ros2
     status: maintained
+  sol_vendor:
+    doc:
+      type: git
+      url: https://github.com/OUXT-Polaris/sol_vendor.git
+      version: main
+    release:
+      tags:
+        release: release/foxy/{package}/{version}
+      url: https://github.com/OUXT-Polaris/sol_vendor-release.git
+      version: 0.0.1-1
+    source:
+      test_pull_requests: true
```

Comment:
@HansRobo just in case, have you [configured your repository](http://wiki.ros.org/buildfarm/Pull%20request%20testing) for this to work?